### DISCUSSION TITLE
Error: robot window invalid library name ; Error: Cannot load the "" robot window library.

### DISCUSSION THREAD

**Discussion by HuNingHe** [cyberbotics/webots#4380] [Q&A]
When I try to open generic robot window,I see this error, And when I try to open generic window, I see "can not find wbw_ini";

webots: version 2022-3-22 webots 2022b-amd.deb on ubuntu20.04

When I haven't installed ros noetic, I can open the robot window normally, but after installation, 
it seems that I can't open it, and how to solve it?

**Comment by omichel** [comment]
This may be related to #4385.

**Reply by HuNingHe** [reply]
Ok thanks for replying
